spotify-auth/functions/index.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const functions = require('firebase-functions');
+const cookieParser = require('cookie-parser');
+const crypto = require('crypto');
+
+// Firebase Setup
+const admin = require('firebase-admin');
+const serviceAccount = require('./service-account.json');
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: `https://${process.env.GCLOUD_PROJECT}.firebaseio.com`
+});
+
+const OAUTH_SCOPES = ['user-read-email'];
+
+/**
+ * Creates a configured Spotify API Client instance.
+ */
+function spotifyClient() {
+  // Spotify OAuth 2 setup
+  // TODO: Configure the `spotify.client_id` and `spotify.client_secret` Google Cloud environment variables.
+  var SpotifyWebApi = require('spotify-web-api-node');
```

Comment:
Change `var` to `const`. Also you could move all this to the static init section :)